packages/report-server/src/reportBuilder/transform/functions/transformValidators.ts

```diff
@@ -29,7 +29,7 @@ export const mapStringToStringValidator = yupTsUtils.describableLazy(
       const stringToStringMapValidator = Object.fromEntries(
         Object.entries(value || {}).map(([field]) => [field, yup.string().required()]),
       );
-      return yup.object().shape(stringToStringMapValidator).required();
+      return yup.object().shape(stringToStringMapValidator).notRequired();
```

Comment:
It will fail this [test ](https://github.com/beyondessential/tupaia/blob/b2690922f5bd084375de17335f6f7bd8e605a2cc/packages/report-server/src/__tests__/reportBuilder/transform/updateColumns.test.ts#L10), when the `insert` is not specified.